### PR TITLE
Updates ARM template files - Service Fabric

### PR DESCRIPTION
**Pull request by davidsmatlak**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

- Updates template used in ARM template [quickstart](https://docs.microsoft.com/azure/service-fabric/quickstart-cluster-template) article.
- Adds quickstart article's author to `docOwner` in _metadata.json_ and copyedits.
- Updates _README.md_ with link to the quickstart article and copyedits.
- Fixes order of template's `parameters` and `resources` properties based on best practices guidance.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@erikadoyle - check this PR for updates that may be needed to documentation that references this sample.

**Comment by erikadoyle** [comment]
Thanks @bmoore-msft. Could we update `"githubUsername": "chackdan"` to **erikadoyle**? Otherwise LGTM. Thanks for cleaning this up @davidsmatlak!

File: service-fabric-secure-cluster-5-node-1-nodetype/metadata.json [hunk 1]
```diff
@@ -2,10 +2,9 @@
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "type": "QuickStart",
   "itemDisplayName": "Deploy a 5 Node Secure Cluster",
-  "description": "This template allows you to deploy a secure 5 node Service Fabric Cluster running Windows Server 2016 Datacenter on a Standard_D2 Size VMSS.",
+  "description": "This template allows you to deploy a secure 5 node Service Fabric Cluster running Windows Server 2019 Datacenter on a Standard_D2_v2 Size VMSS.",
   "summary": "This template takes a minimum amount of parameters and deploys 5 Node Service Fabric Cluster",
   "githubUsername": "chackdan",
```
Comment (erikadoyle) [inline service-fabric-secure-cluster-5-node-1-nodetype/metadata.json:7]: Could we update githubUsername to **erikadoyle** ? Thanks!

**Review by erikadoyle — COMMENTED** [review]
Just an update to githubUsername, otherwise looks great.

**Comment by davidsmatlak** [comment]
Hi @erikadoyle thanks for the feedback. I'll update the `githubUserName`.

File: service-fabric-secure-cluster-5-node-1-nodetype/metadata.json (hunk 1 above)
Comment (davidsmatlak) [reply service-fabric-secure-cluster-5-node-1-nodetype/metadata.json:7]: Fixed.

**Comment by bmoore-msft** [comment]
@erikadoyle - check this PR for updates that may be needed to documentation that references this sample.

File: service-fabric-secure-cluster-5-node-1-nodetype/metadata.json (hunk 1 above)
Comment (davidsmatlak) [reply service-fabric-secure-cluster-5-node-1-nodetype/metadata.json]: @erikadoyle I noticed the [metadata](https://github.com/azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#metadatajson) guidance states `githubUsername` is for the original template author and shouldn't be changed. I'll revert the change.

**Comment by bmoore-msft** [comment]
@erikadoyle - check this PR for updates that may be needed to documentation that references this sample.

**Comment by bmoore-msft** [comment]
@erikadoyle - check this PR for updates that may be needed to documentation that references this sample.

**Comment by erikadoyle** [comment]
LGTM. The github user *chackdan* is no longer at the company, but I'll assume we retain it to preserve author history.

**Comment by bmoore-msft** [comment]
We can change the author - just need to have someone willing to pick up ownership of it...

**Comment by davidsmatlak** [comment]
@bmoore-msft thanks for the update. 

@erikadoyle I'll submit a pull request to change the `githubUsername`.

**Pull request merged**